RequestManagementSystem/DB/RequestDB.py

```diff
@@ -517,7 +517,7 @@ def getRequestIDsList( self, statusList = None, limit = None, since = None, unti
 
       reqQuery = reqQuery.order_by( Request._LastUpdate )\
                          .limit( limit )
-      requestIDs = [reqIDTuple[0] for reqIDTuple in reqQuery.all()]
+      requestIDs = [ tuple( reqIDTuple ) for reqIDTuple in reqQuery.all() ]
```

Comment:
This is how it is used now in various places:
res = reqClient.getRequestIDsList( [status], limit = 999999999, since = since, until = until )

    if not res['OK']:
      gLogger.error( "Error getting requests:", res['Message'] )
      DIRAC.exit( 2 )
    requests = [reqID for reqID, _st, updTime in res['Value'] if updTime > since and updTime <= until and reqID]